the select_add example panics

Eg

```
~/go/src/github.com/manifoldco/promptui/_examples/select_add$ go run main.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x54d8f8]

goroutine 1 [running]:
github.com/manifoldco/promptui.(*Select).innerRun.func1(0x0, 0x0, 0x0, 0x0, 0xc400000000, 0x50c75c, 0xc420093260, 0xc42007ec60, 0xc4200bba60, 0xc42007ec60)
	/home/ncw/go/src/github.com/manifoldco/promptui/select.go:159 +0x78
github.com/chzyer/readline.(*DumpListener).OnChange(0xc42000e0d8, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc4200bbb10, 0x48f40b, 0xc42000e018, 0x0, ...)
	/home/ncw/go/src/github.com/chzyer/readline/operation.go:516 +0x63
github.com/chzyer/readline.(*Operation).Runes(0xc420093260, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/ncw/go/src/github.com/chzyer/readline/operation.go:385 +0x37e
github.com/chzyer/readline.(*Operation).String(0xc420093260, 0x8, 0x583860, 0x1, 0xc42000e0d8)
	/home/ncw/go/src/github.com/chzyer/readline/operation.go:375 +0x2b
github.com/chzyer/readline.(*Instance).Readline(0xc42000c5c0, 0xc42000e0d8, 0x6, 0x8, 0x6)
	/home/ncw/go/src/github.com/chzyer/readline/readline.go:255 +0x2f
github.com/manifoldco/promptui.(*Select).innerRun(0xc4200621e0, 0x1, 0x2b, 0xc42000c400, 0x0, 0x0, 0x5, 0x5)
	/home/ncw/go/src/github.com/manifoldco/promptui/select.go:257 +0x320
github.com/manifoldco/promptui.(*SelectWithAdd).Run(0xc420072280, 0xc4200722d0, 0x56f220, 0x5cc790, 0xc420018430, 0xb)
	/home/ncw/go/src/github.com/manifoldco/promptui/select.go:421 +0x45e
main.main()
	/home/ncw/go/src/github.com/manifoldco/promptui/_examples/select_add/main.go:16 +0x158
exit status 2
```

--- a/select.go
+++ b/select.go
@@ -412,6 +412,7 @@ func (sa *SelectWithAdd) Run() (int, string, error) {
 			Size:      5,
 			list:      list,
 		}
+		s.setKeys()
 
 		err = s.prepareTemplates()
 		if err != nil {

